SU-HSDO/suhumsci
# shs-122: update accordion state to use correct mixin

## Summary
Fixes an issue with border colors on the alternative accordion presentation. The original mixin does not respect color pairings, only a pairing for the theme's default color theme.

## Steps to Test
1. Go to the accordions page (`/accordions`). Edit the layout, under the Main Region, add the `hb-accordion-secondary` class to the Field Formatter Class area.
2. Visit the accordion page and verify that it's using the correct layout (border color of the theme's secondary color, hover is the background primary color etc)
3. Go to the theme settings (`/admin/appearance/settings/humsci_colorful`) under Color Pairing change the color to a different color. Go back to the accordions page and verify it changed to the appropriate color. Do this a few times to be sure.
4. Repeat these steps for the Traditional site.

## Screenshots
![Screen Shot 2022-11-02 at 2 28 49 PM](https://user-images.githubusercontent.com/8405274/199573342-b1a8d0c9-3daa-4931-8771-a4cee3efa7ae.png)
